### DISCUSSION TITLE
Cannot access Student Developer pack after 1 month of approval

### DISCUSSION THREAD

**Discussion by hlzswbc** [community/community#72306] [GitHub Education]
### Hi everyone,

Hi folks, do you know how long it would take for a student to access the student pack? I am eager to get the developer pack but after one month of approval (was approved on 09/30/2023) I have yet received any email about the student pack

**Comment by ktauchathuranga** [comment]
i saw a similar discussion.
here, refer [this](https://github.com/orgs/community/discussions/66773#discussioncomment-6968652)
hope this helps 🖤

**Reply by hlzswbc** [reply]
Ah yeah looks like many people have the same situation as mine. Thanks for sharing this!
